docs/features/web-api-template.md

```diff
@@ -39,6 +39,9 @@ And additional features available with options:
   * `None`: no authentication configured on the API project.
 * `-ec|--exclude-correlation` (default `false`): excludes the [capability to correlate](https://webapi.arcus-azure.net/features/correlation) between HTTP requests/responses.
 * `-ia|--include-appsettings` (default `false`): includes a `appsettings.json` file to the web API project.
+* `-lo|--logging` (default `Default`)
+  * `Default`: no extra logging mechanism except for the default console logging will be added to the web API project.
```

Comment:
`Default`, `Minimal`, `Console` or `Basic`?

Reply:
Hmm, 'Console' would be better indeed.